.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+	"singleQuote": true,
+	"jsxSingleQuote": false,
+	"trailingComma": "all",
+	"printWidth": 140,
+	"semi": true,
+	"arrowParens": "always",
+	"useTabs": true,
+	"bracketSpacing": true,
+	"jsxBracketSameLine": false
+}
```

Comment:
Remove this - we are not using prettier in this repo.

Reply:
See https://github.com/NoA-Ignite-dk/eslint-config/pull/14#issuecomment-1056517507